addons/survey/views/survey_templates_statistics.xml

```diff
@@ -263,56 +311,70 @@
 
     <template id="question_result_choice" name="Question: choice result (simple_choice, multiple_choice)">
         <ul class="nav nav-tabs d-print-none" role="tablist">
-            <li class="nav-item">
-                <a t-att-href="'#survey_graph_question_%d' % question.id"
-                    t-att-aria-controls="'survey_graph_question_%d' % question.id"
-                    class="nav-link active" data-toggle="tab" role="tab">
-                    <i class="fa fa-bar-chart-o"></i>
-                    <span t-if="question.question_type == 'simple_choice'">Pie Graph</span>
-                    <span t-else="">Graph</span>
-                </a>
-            </li>
+            <t t-if="question_answered">
```

Comment:
You can put the "t-if" on the actual `<li>` element to avoid extra indentation / diff.